backend/src/middlewares/authMiddleware.ts

```diff
@@ -0,0 +1,23 @@
+import { UNAUTHORIZED_ERROR } from '../auth/index';
+import { RequestHandler, Response } from "express";
+import { verifyToken } from "../auth";
+
+const sendError = (res: Response, errorMessage: string) => {
+    res.statusCode = 400;
```

Comment:
You know that not all errors should be 400? There is a reason there are dozens of them. 400 means bad request but sometimes you should return 500, 404, 403 all of which has meaning.